### ISSUE TITLE
RES generators are relocated over a long distance

### ISSUE THREAD

**Issue by IlkaCu** [openego/data_processing#277]
![grafik](https://user-images.githubusercontent.com/16799094/41172309-c3f38d1c-6b53-11e8-8fa8-8ce218733feb.png)

As the screenshot shows some RES generators which are relocated over a very long distance. The geom of the affected generators was originally placed outside the German territory/ the mv grid districts. I'm pretty sure that we already dealt with the problem of RES generators which were placed outside the German borders, but I cannot find the corresponding github issue?! 

Can someone help?

**Comment by IlkaCu** [comment]
In total 62479 generators are affected. I noticed this because for those generators the assignment of an subst_id in script ego_dp_substation_id_to_generator.sql based on their geom fails. 
I assume that the REA algorithmn works and the problem was caused by the fact that we didn't clear the column 'subst_id' in the beginning of ego_dp_substation_id_to_generator.sql. 
Now I wonder whether the REA scripts can handle RES generators located outside the MV griddistricts or not. Even if they do, we need to adjust the DP to include the subst_id for the affected generators.
